frontend/src/components/modals/pointPopup/PointPopup.tsx

```diff
@@ -0,0 +1,291 @@
+import React, { useState } from 'react';
+import { styled } from '@mui/material/styles';
+import IconButton, { IconButtonProps } from '@mui/material/IconButton';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import ReportIcon from '@mui/icons-material/Report';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import CloseIcon from '@mui/icons-material/Close';
+import IosShareIcon from '@mui/icons-material/IosShare';
+import ThumbUpIcon from '@mui/icons-material/ThumbUp';
+import ThumbDownIcon from '@mui/icons-material/ThumbDown';
+import CheckCircleTwoToneIcon from '@mui/icons-material/CheckCircleTwoTone';
+import SendIcon from '@mui/icons-material/Send';
+import {
+  Card,
+  CardHeader,
+  CardMedia,
+  CardContent,
+  CardActions,
+  Collapse,
+  Avatar,
+  Typography,
+  Box,
+  TextField,
+  FormControl,
+  Button,
+  List,
+  ListItem,
+  Divider,
+  ListItemText,
+  ListItemAvatar
+} from '@mui/material';
+
+import './pointPopup.scss';
+
+// It can be reused in different components
+// When the rest of the functionality is available, this component will be assigned to a specific and each point on the map
+// Change position a photo, description, buttons, etc. you can without problems
+// Now it works on stubs
+
+interface Props {
+  active: boolean;
+  setActive: Function;
+  children: Array<any>;
+}
+
+interface ExpandMoreProps extends IconButtonProps {
+  expand: boolean;
+}
+
+const ExpandMore = styled((props: ExpandMoreProps) => {
+  const { expand, ...other } = props;
+  /* eslint-disable-next-line react/jsx-props-no-spreading */
+  return <IconButton {...other} />;
+})(({ theme, expand }) => ({
+  transform: !expand ? 'rotate(0deg)' : 'rotate(180deg)',
+  marginLeft: 'auto',
+  transition: theme.transitions.create('transform', {
+    duration: theme.transitions.duration.shortest
+  })
+}));
```

Comment:
You can place it inside _styles.ts_. But it's up to you.

Reply:
I'll fix it.